languages/purescript/config.json

```
@@ -12,8 +12,14 @@
       {
         "slug": "booleans",
         "uuid": "f1eb2cab-d6bc-447a-bce3-6780c44e4b22",
-        "concepts": ["booleans"],
+        "concepts": ["booleans", "basics"],
         "prerequisites": []
+      },
+      {
+        "slug": "pattern-matching",
+        "uuid": "74cc435c-403a-428b-82e6-9038599b7633",
+        "concepts": ["pattern-matching"],
+        "prerequisites": ["strings", "conditionals", "basics"]
```

If an exercise has another prerequisite besides `basics`, the `basics` concept can be omitted (see [this section](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)):

```suggestion
        "prerequisites": ["strings", "conditionals"]
```